hints/sliding-window-maximum.md

```diff
@@ -0,0 +1,39 @@
+<br>
+<details class="hint-accordion">  
+    <summary>Recommended Time & Space Complexity</summary>
+    <p>
+    You should aim for a solution with <code>O(nlogn)</code> time and <code>O(n)</code> space, where <code>n</code> is the size of the input array.
```

Comment:
I think we should say "as good or better" since an O(n) solution exists. But let me know if you did this for a reason.

Reply:
Sorry, it was my mistake. But I can assure you that, I always try to see the work I have done neetcode and I always try to correct that If I made any mistake. This mistake you mentioned is an example.

Reply:
I also got to know that I have done some mistakes in the solutions, when I see that I corrected those articles when I was writing Go and Kotlin solutions. Now I am 100% sure that there is no solution in those articles that gives a wrong answer and any errors other than TLE for brute force.

Reply:
@Srihari2222 Thank you, i appreciate that.